ecs.tf

```diff
@@ -93,6 +101,25 @@ resource "aws_s3_bucket_object" "prometheus_config" {
   kms_key_id = data.terraform_remote_state.management.outputs.config_bucket.cmk_arn
 }
 
+resource "aws_service_discovery_private_dns_namespace" "prometheus" {
+  name = "services.${var.parent_domain_name}"
+  vpc  = module.vpc.outputs.vpc_ids[0]
+}
+
+resource "aws_service_discovery_service" "prometheus" {
+  count = length(local.roles)
+  name  = local.roles[count.index]
```

Comment:
Maybe interpolate this with `var.name` as you do in line 84 so we don't end up with services simply named "master" and "slave" but instead "monitoring-master" and "monitoring-slave"?

Reply:
Resolved